timesketch/lib/analyzers/phishy_domains.py

```diff
@@ -41,8 +41,8 @@ def __init__(self, index_name, sketch_id):
 
         self.domain_scoring_threshold = current_app.config.get(
             'DOMAIN_ANALYZER_WATCHED_DOMAINS_SCORE_THRESHOLD', 0.75)
-        self.domain_scoring_whitelist = current_app.config.get(
-            'DOMAIN_ANALYZER_WHITELISTED_DOMAINS', [])
+        self.domain_scoring_allowlist = current_app.config.get(
```

Comment:
Can't come up with anything rightnow, but this one need to reflect what it is. Some other name.. I'll get back on that

Reply:
DOMAIN_ANALYZER_DONOTCHECK_DOMAINS would be what this means the most maybe?

Reply:
> Yes, good point. Let's get rid of the term whitelist. Allow list is not necessarily the right term for all of these though. It's context sensitive. I'll add in comments in the code where I think we should name it something else.
> 
> We also need to take care of not breaking folks who have a config file already. Please add a warning to logs saying that the config parameter "whitelist" has been deprecated and urge folks to update their config file.

Just to verify I understand correctly, do you suggest a warning for people who update should get a warning, or when someone tries to get a "new" value but has an "old" config.

The first I would do in the setup.py. The later would be an if clause at every call that I changed. Or do we have a better way to "age" configs? If there is a PR that did something similar, happy to copy that.

Reply:
So, we rarely change old things in the config (maybe never?), just add new things.
If we rename this config parameter, it will break installations that have an old version of the config. 

We need to warn on this in the __init__.py file, where all the Flask init stuff happens.

Reply:
> DOMAIN_ANALYZER_NOCHECK_DOMAINS would be what this means the most maybe?

DOMAIN_ANALYZER_EXCLUDE_DOMAINS
and
self.domain_scoring_exclude_domains

WDYT?

Reply:
ACK; I will give it a try and come up with a warning etc.

Would it be an idea to introduce a version number in the config file that we can use looking forward to check such things?

Reply:
Yeah, it's complicated :) The config file is meant to be copied by the server admin, and it could in theory contain additions etc. Also, when new stuff is introduced to the standard config most folks will just copy it over to their old config file.

So I think that a version on the config isn't worth it.

Reply:
So, you need to do a check in all places where the config variable is used, and warn there, telling the user that it has been deprecated. You need to support both names for a while.